packages/expo-doctor/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "expo-doctor",
+  "version": "1.1.3",
+  "main": "build/index.js",
+  "description": "Check your Expo project for known issues",
+  "keywords": [
+    "expo",
+    "ios",
+    "expo-doctor",
+    "android"
+  ],
+  "homepage": "https://docs.expo.dev",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git",
+    "directory": "packages/expo-doctor"
+  },
+  "author": "Expo <support@expo.dev>",
+  "license": "MIT",
+  "bin": {
+    "expo-doctor": "./bin/expo-doctor"
+  },
```

I would recommend just using this instead, but ofc not a hard requirement to merge!
```suggestion
  "bin": "./bin/expo-doctor",
```